cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -150,6 +150,21 @@ function wait_pods_ready {
     done
 }
 
+function wait_pods_condition_by_label {
+  local -r tries=20
+  local -r wait_time=3
```

Comment:
It costs now 20*30sec + 20*3sec = 660sec = 11min
You added another minute in case of failure.

Reply:
Just in case of absolute failure, which rarely happens, but now is protected as well.
In case of success, even with pod restart it saves a lot of time.

Usually it takes just 1 minute actually locally, because it continue once condition is met
i just didn't want to play with the times
I will change it to wait_time=1, and tries=15 i believe it will be totally safe.

Reply:
Well, changing the time diverges from the main objective of this change.
But 10m is also a very long time, so it makes sense to reduce it.

Anyway, whatever you do, you need to mention this in the commit message and PR description so it will be clear this is intentional and you need to reason for it.